composer.json

```diff
@@ -58,7 +58,7 @@
         "contao-components/tablesorter": "^2.1",
         "contao-components/tinymce4": "^5.0 || ^6.0",
         "contao-components/tristen-tablesort": "^5.3",
-        "contao/image": "^1.0",
+        "contao/image": "1.x-dev",
```

Comment:
Any chance that we can release a stable version before merging this PR?

Reply:
Yes we should. But I’d like to have more feedback from the community first, see https://github.com/contao/image/issues/80#issuecomment-1435661044